pkg/logs/status/status_test.go

```diff
@@ -70,3 +71,8 @@ func TestStatusDeduplicateWarnings(t *testing.T) {
 	status := Get()
 	assert.ElementsMatch(t, []string{"Identical Warning", "Unique Warning"}, status.Messages)
 }
+
+func TestMetrics(t *testing.T) {
```

Comment:
Can you add more tests please to assert that:
- adding new messages increase the number of warnings
- setting up the agent change the value of `IsRunning`

Reply:
done